ansible/roles/celery/tasks/main.yml

```diff
@@ -4,10 +4,25 @@
   sudo: yes
   tags: celery
 
+- name: Fetch GPG key for docker package
```

Comment:
There's an [``apt-key`` module for Ansible](https://docs.ansible.com/ansible/apt_key_module.html). Here's an [example usage](https://github.com/ImageMarkup/isic-archive/blob/master/ansible/roles/mongodb/tasks/main.yml#L3).